test/FunctionalTests/Tests/DataServices/UnitTests/ClientExtensions/DataServiceContextWithCustomTransportLayer.cs

```diff
@@ -65,7 +65,6 @@ public DataServiceContextWithCustomTransportLayer(Uri serviceUri, ODataProtocolV
         internal DataServiceClientRequestMessage CreateMessage(DataServiceClientRequestMessageArgs requestMessageArgs)
         {
             var requestMessage = this.getRequestMessage();
-            requestMessage.Url = requestMessageArgs.RequestUri;

```

Comment:
We don't need to do this anymore?

Reply:
Hmm. Not sure why I removed this. I've put it back.